src/Palettes.md

```diff
@@ -22,72 +22,89 @@ Value | Color
   2   | Dark gray
   3   | Black
 
-In CGB Mode the Color Palettes are taken from [CGB Palette Memory](<#LCD Color Palettes (CGB only)>)
+In CGB Mode the color palettes are taken from [CGB palette memory](<#LCD Color Palettes (CGB only)>)
 instead.
 
-### FF48 - OBP0 (Object Palette 0 Data) (R/W) - Non CGB Mode Only
+### FF48 - OBP0 (OBJ Palette 0 Data) (R/W) - Non CGB Mode Only
 
-This register assigns gray shades to the color indexes of the OBJs that use this palette. It works exactly
-like BGP (FF47), except that the lower two bits are ignored because
-sprite index 00 means transparent.
+This register assigns gray shades to the color indexes of the OBJs that use
+this palette. It works exactly like BGP, except that the lower two bits are
+ignored because color index 0 is transparent for OBJs.
 
-### FF49 - OBP1 (Object Palette 1 Data) (R/W) - Non CGB Mode Only
+### FF49 - OBP1 (OBJ Palette 1 Data) (R/W) - Non CGB Mode Only
 
-This register assigns gray shades to the color indexes of the OBJs that use this palette. It works exactly
-like BGP (FF47), except that the lower two bits are ignored because
-sprite index 00 means transparent.
+This register assigns gray shades to the color indexes of the OBJs that use
+this palette. It works exactly like BGP, except that the lower two bits are
+ignored because color index 0 is transparent for OBJs.
 
 ## LCD Color Palettes (CGB only)
 
+The CGB has a small amount of RAM used to store its color palettes. Unlike most
+of the hardware interface, palette RAM (or *CRAM* for *Color RAM*) is not
+accessed directly, but instead through the following registers:
+
 ### FF68 - BCPS/BGPI (Background Color Palette Specification or Background Palette Index) - CGB Mode Only
 
-This register is used to address a byte in the CGBs Background Palette
-Memory. Each two byte in that memory define a color value. The first 8
-bytes define Color 0-3 of Palette 0 (BGP0), and so on for BGP1-7.
+This register is used to address a byte in the CGB's background palette RAM.
+Since there are 8 palettes, 8 palettes × 4 colors/palette × 2 bytes/color = 64 bytes
+can be addressed.
 
 ```
 Bit 7     Auto Increment  (0=Disabled, 1=Increment after Writing)
-Bit 5-0   Index (00-3F)
+Bit 5-0   Address ($00-3F)
 ```
 
-Data can be read/written to/from the specified index address through
-Register FF69. When the Auto Increment bit is set then the index is
-automatically incremented after each **write** to FF69. Auto Increment has
-no effect when **reading** from FF69, so the index must be manually
-incremented in that case. Writing to FF69 during rendering still causes
-auto-increment to occur.
+First comes BGP0 color #0, then BGP0 color #1, BGP0 color #2, BGP0 color #3,
```

Comment:
Why are we using `#` for "number" here? Is is something we do somewhere else ?

Reply:
No, according to some research, spelling out "number" or maybe `n°`?

Reply:
I feel like `#' can be used but it must be a document-wise thing. In the mean time, both of the following suggestions would work for me

Reply:
I want to avoid the former because of potential ambiguity.